playground/nuxt.config.ts

```diff
@@ -2,13 +2,21 @@ export default defineNuxtConfig({
   modules: ['../src/module', '@nuxt/devtools'],
   ssr: true,
 
-  compatibilityDate: '2024-07-17',
+  build: {
```

Comment:
Hi @justserdar this config is redundant since the module already handles it here https://github.com/Tresjs/nuxt/blob/2a825cb2a730d764a45767197a794c48b9d9f57a/src/module.ts#L30, please remove it

Reply:
Hey @alvarosabu Thanks for taking the time to check the PR, I was under the impression the PR's were scrapped.
I removed the build section from the playground config.